### PR TITLE
Updated to suit the latest changes

### PR DESCRIPTION
**Pull request by Incognitov**
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/huytd%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613943%3Fv%3D3%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/373%23issuecomment-172971886%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-01-19T20%3A10%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613943%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/huytd%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20f534b6ddd91bd9bafa465c763c0cff6dcf046cc4%20README.md%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/huytd/agar.io-clone/pull/373%23discussion_r50167149%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22ah...%20great...%20haha%22%2C%20%22created_at%22%3A%20%222016-01-19T20%3A06%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613943%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/huytd%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222016-01-19T20%3A12%3A22Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/613943%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/huytd%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20README.md%3AL5-48%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/huytd/agar.io-clone/pull/373%23issuecomment-172971886%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/373%23discussion_r50167149%22%2C%20%22https%3A//github.com/huytd/agar.io-clone/pull/373%23discussion_r50168062%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/huytd'></a>

- [x] <a href='#crh-comment-Pull f534b6ddd91bd9bafa465c763c0cff6dcf046cc4 README.md 9'></a> &nbsp;<b><a href='https://github.com/huytd/agar.io-clone/pull/373#discussion_r50167149'>File: README.md:L5-48</a></b>
- <a href='https://github.com/huytd'></a> ah... great... haha

<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/373?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/373?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/huytd/agar.io-clone/pull/373?approve=0'></a>&nbsp;<a href='https://github.com/huytd/agar.io-clone/pull/373'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by huytd** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/huytd/agar.io-clone/pull/373'>Refresh</a>.<a href='#crh-update-none'></a>

File: README.md [hunk 1]
```diff
@@ -5,41 +5,43 @@ Agar.io Clone
 [![GitHub Issues](https://img.shields.io/github/issues/huytd/agar.io-clone.svg)](https://github.com/huytd/agar.io-clone/issues)
 [![GitHub Wiki](https://img.shields.io/badge/project-wiki-ff69b4.svg)](https://github.com/huytd/agar.io-clone/wiki/Home)
 ![Build Status](https://api.travis-ci.org/huytd/agar.io-clone.svg)
-[![Live Demo](https://img.shields.io/badge/demo-online-green.svg)](#live-demo) "
+[![Live Demo](https://img.shields.io/badge/demo-online-green.svg)](#live-demo) -
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/huytd/agar.io-clone?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-A simple Agar.io clone built with socket.io and HTML5 Canvas on top of NodeJS.
+A simple but powerful Agar.IO clone built with socket.IO and HTML5 canvas on top of NodeJS.
```
Comment (huytd) [inline README.md]: ah... great... haha

**Comment by huytd** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (huytd) [inline README.md]: :+1:

**Comment by IgorAntun** [comment]
:shipit:

**Pull request merged**